spec/features/user_can_see_specific_book_spec.rb

```diff
@@ -1,14 +1,17 @@
 require 'rails_helper'
 
-describe 'as a user i can see a specific book and all reviews for it' do
-  describe 'when i am on the book show path' do
-    scenario 'then i can see book title, reviews' do
-      book = Book.create!(title: "The Jungle Book")
-      user = User.create!(name: "Bob Ross")
-      review = user.reviews.create!(body: "Made me feel warm and fuzzy", rating: 5, book: book)
+describe 'as a user' do
+  describe 'when im on the book show page' do
+    scenario 'then i see book name, review body and usser that created review and ratings' do
+      book = Book.create!(title: "Age of Ultron")
+      user = User.create!(name: "tony Stark")
+      review = book.reviews.create!(body: "this books sucked", rating: 2, user: user)
+
       visit book_path(book)
 
-      expect(page).to have_content("The Jungle Book")
+      expect(page).to have_content("Age of Ultron")
+      expect(page).to have_content("this books sucked")
+      expect(page).to have_content(2)
```

Comment:
I'd rather see here book.title rather than "The Jungle Book". I have to do less work with the first one to figure out the context of what you're expecting.